MoMMI/Modules/github.py

```diff
@@ -790,6 +780,32 @@ def gh_register_help() -> None:
 
         await post_embedded_issue_or_pr(channel, repo, rand_issue)
 
+@command("autolabels", r"(?:(\S+)#)?(?:autolabels|autolabel)")
+async def autolabels_command(channel: MChannel, match: Match, message: Message) -> None:
+    prefix = match.group(1)
+
+    for repo_config in cfg:
+        repo = repo_config["repo"]
+
+        if not is_repo_valid_for_command(repo_config, channel, prefix):
+            continue
+
+        autolabels: Dict[str, str] = master.config.get_module(
+                f"github.repos.{repo}.autolabels", {})
+        if not autolabels:
+            await master.client.add_reaction(message, "❌")
+        else:
+            await master.client.add_reaction(message, "⏳")
```

Comment:
Don't bother with these embeds. You will literally spend more time sending the reactions than actually sending the data.